brave/brave-core
# Refactor balance report

Resolves https://github.com/brave/brave-browser/issues/7329

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
plan 1:
- enable rewards
- claim grant
- do one time tip
- make sure that it's shown in panel/settings page wallet summary
- check if it's displayed on new tab page

plan 2:
- go to master
- do plan 1
- switch to this PR
- start browser
- make sure that data on settings page, panel and new tab page are correct

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
